e2-studio-star-rx72n-firmware/tests/mocks/mock_rx_hcsr04_irq.c

```
@@ -0,0 +1,57 @@
+{
+  (void)pin;
+  return k_rx_ok;
+}
+
+rx_err_t rx_mpc_set_irq(rx_port_pin_t pin)
+{
+  (void)pin;
+  return k_rx_ok;
+}
+
+rx_err_t rx_hcsr04_icu_configure(uint8_t irq_num, uint8_t priority)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_hcsr04_icu_configure' of similar type ('uint8_t') are easily swapped by mistake

(bugprone-easily-swappable-parameters)